corehq/apps/fixtures/dispatcher.py

```diff
@@ -19,4 +19,5 @@ class FixtureInterfaceDispatcher(ProjectReportDispatcher):
 
     @method_decorator(require_can_edit_fixtures)
     def dispatch(self, request, *args, **kwargs):
+        request.use_requirejs = True
```

Comment:
i'm actually OK with this. it's how we turn on and off a lot of things in HQ so it fits within that style. minor point is that i think we usually use a decorator though